internal/app/agent/service.go

```diff
@@ -135,30 +169,30 @@ func New(mc paho.Client, cfg *config.Config, ec edgex.Client, nc *nats.Conn, log
 
 }
 
-func (a *agent) Execute(uuid, cmd string) (string, error) {
+func (a *agent) Execute(uuid, cmd string) (string, errors.Error) {
```

Comment:
With new `errors.Error` there is no need to change signatures of methods, you can keep just `error`. And you can still use `errors.Wrap`, because now all arguments and returning value is generic error type (`func Wrap(wrapper error, err error) error`) . You don't have to use Wrapping everywhere, just where you think it has sence.